src/components/TodoContainer.jsx

```diff
@@ -0,0 +1,38 @@
+import React, { useState, useEffect } from "react";
+import "../styles/TodoContainer.css";
+import TodoInput from "./TodoInput";
+import TodoList from "./TodoList";
+
+const TodoContainer = () => {
+  const [todos, setTodos] = useState([]);
+
+  useEffect(() => {
+    getLocalTodos();
+  }, []);
+
+  useEffect(() => {
+    saveLocalTodos();
+  });
+
+  const saveLocalTodos = () => {
+    localStorage.setItem("todos", JSON.stringify(todos));
+  };
+
+  const getLocalTodos = () => {
+    if (!localStorage.getItem("todos")) {
+      localStorage.setItem("todos", JSON.stringify([]));
+    } else {
+      const localTodos = JSON.parse(localStorage.getItem("todos"));
+      setTodos(localTodos);
+    }
+  };
```

Comment:
Hi! @ahmedeid6842 good job! I like the UI and the integration with the localStorage to save the To-Do's, the perform of your code is very clear and clean.